docs/developers/guides/geth_js_console.md

```diff
@@ -0,0 +1,166 @@
+# Geth JavaScript Console Guide
+
+This is a comprehensive guide to setting up and using the Geth JS console with an Evmos node.
+
+## ✅ Requirements
+
+- Geth
+- Evmos (local) node
```

Comment:
Could you add links to these list items? For instance, a link to `geth` documentation and the link to the documentation which shows how to start the local node. Afterward, append `{prereq}` to the end of each list item (like the `{synopsis}` above).